[en] add wsbind permission (#4911)
DaoCloud/DaoCloud-docs: docs/en/docs/ghippo/user-guide/workspace/wsbind-permission.md

> # Resource Binding Permission Instructions

Reviewer: 可以增加 MTPE front matter

---
MTPE: WANG0608GitHub
date: 2024-07-16
---

# Resource Binding Permission Instructions

> If user John ("John" represents any user with resource binding needs) already has 

Reviewer: ```suggestion
If a user John ("John" represents any user who is required to bind resources) has 
```

> the [Workspace Admin role](../access-control/role.md#_4) or has been granted it through a [custom role](../access-control/custom-role.md), 

Reviewer: ```suggestion
the [Workspace Admin role](../access-control/role.md#_4) assigned or has been granted proper permissions through a [custom role](../access-control/custom-role.md), 
```

> which includes the [Workspace's "Resource Binding" Permissions](./ws-permission.md#_3), and wishes to bind a specific cluster or namespace to the workspace.

Reviewer: ```suggestion
which includes the [Workspace's "Resource Binding" Permissions](./ws-permission.md#_3), and wants to bind a specific cluster or namespace to the workspace.
```

> To bind cluster/namespaces resources to Workspace, not only the [Workspace's "Resource Binding" Permissions](./ws-permission.md#_3) is required,

Reviewer: ```suggestion
To bind cluster/namespace resources to a workspace, not only the [workspace's "Resource Binding" permissions](./ws-permission.md#_3) are required,
```
`.md#_3` 锚点要替换为真实的英文锚点

>  but also the resource permission of [Cluster Admin](../../../kpanda/user-guide/permissions/permission-brief.md#cluster-admin).

Reviewer: ```suggestion
but also the permissions of [Cluster Admin](../../../kpanda/user-guide/permissions/permission-brief.md#cluster-admin).
```

## Granting Authorization to John

1. Using the [Platform Admin Role](../access-control/role.md#_2),

>    grant John the Workspace Admin role on the **Workspace** -> **Authorization** page.

Reviewer: ```suggestion
   grant John the role of Workspace Admin on the **Workspace** -> **Authorization** page.
```

    ![Resource Binding](../../images/wsbind1.png)

1. Then, on the **Container Management** -> **Permissions** page, authorize John as a Cluster Admin by **Add Permission**.

    ![Cluster Permissions1](../../images/wsbind2.png)

    ![Cluster Permissions2](../../images/wsbind3.png)

## Binding to Workspace

Using John's account to log in to DCE 5.0, on the **Container Management** -> **Clusters** page, 

>  John can bind the specified cluster to his own workspace through the **Bind Workspace** function

Reviewer: ```suggestion
John can bind the specified cluster to his own workspace by using the **Bind Workspace** button.
```

!!! note

    John can only bind clusters or namespaces to a specific workspace in the [Container Management module](../../../kpanda/intro/index.md), and cannot perform this operation in the Global Management module.

![cluster banding](../../images/wsbind4.png)

> Binding a namespace to a workspace also requires at least Workspace Admin + Cluster Admin permissions.

Reviewer: ```suggestion
To bind a namespace to a workspace, you must have at least Workspace Admin and Cluster Admin permissions.
```
